tests/foreman/smoke/test_api_smoke.py

```diff
@@ -879,22 +879,24 @@ def test_smoke(self):
         ).create()
 
         # step 2.8: Associate YUM repository to new content view
-        content_view.set_repository_ids([repo1.id])
+        content_view.repository = [repo1]
```

Comment:
I believe we're replacing whatever repositories the content view already has. You can append with something like this:

```python
content_view = ContentView(...).read()
content_view.repository.append(second_repo)
content_view = content_view.update(['repository'])
```